src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -119,6 +129,8 @@ const ContentMessageComponent: React.FC<ContentMessageProps> = ({
     return !previousMessage.isContent() || previousMessage.user().id !== message.user().id;
   };
 
+  const timeAgo = useRelativeTimestamp(message.timestamp());
+
   // check if current message is focused and its elements focusable
```

Comment:
Is this one still in "TODO"?

Reply:
looks like when Adrian refactored message focus state to use enum and memo he forgot to remove the comment

Reply:
i am placing it in the current spot